generators/entity/templates/server/src/main/java/package/domain/_Entity.java

```diff
@@ -58,8 +58,10 @@
 <% } -%>
 <% if (databaseType == 'sql') { -%>
 @Entity
-@Table(name = "<%= entityTableName %>")<% if (hibernateCache != 'no') { %>
-@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)<% } %><% } %><% if (databaseType == 'mongodb') { %>
+@Table(name = "<%= entityTableName %>")
+<%_ if (hibernateCache !== 'no') { if (hibernateCache === 'infinispan') { _%>
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE) <%_ } else { _%>
```

Comment:
Using slurp scriptlet should always be in newline else it will create bad formatted output during generation